lethal-squirrels/Bangazon-Site
# Ajdel

Ticket: #13,#18,#19

DESCRIPTION

I changed layout of the products that show up on the home page
The user can only delete a product if it is not on a completed order
When you click on the MyProducts page, only your products should show up

STEPS TO TEST

Clone ajdel to a local repo

Test functionality
Click on MyProducts link in nav bar
Make sure only the products you have added show up on this page

 When you try and delete a product, it should only let you delete a product if it has never been 
 on a completed order. So the three scenarios to test would be:
a product that is not added to an order - can be deleted - no entry in prodorder or order table
a product added to an incomplete order - can be deleted - entry in prodorder table and order table , with paymenttypeid null in order table
a product added to a complete order - cannot be deleted - entry in prodorder table and order table, with paymenttypeid not null in order table

 On the Home Page, the layout should like the other pages for Products with links to Details et 
 al.
